code/index.html

```diff
@@ -2,21 +2,163 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>Arctic News by Ekaterina Klimenko</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    
+    <header class="header-general">
+      <!-- Header phone: hamburger menu and dropdown navbar -->
+      <div class="header-phone">
+          <img src="./Arctic-Logo.png" class="logo" alt="Arctic News">
+          <div class="checkbox">
+            <input type="checkbox" id="hamburger">
+              <nav class="dropdown">
+                <ul class="dropdown-list">
+                    <li><a href="" class="nav-link">Home</a></li>
+                    <li><a href="" class="nav-link">Highlights</a></li>
+                    <li><a href="" class="nav-link">News</a></li>
+                    <li><a href="" class="nav-link">More</a></li>
+                </ul>
+              </nav>
+            <label class="label" for="hamburger">
+              <div class="hamburger-line"></div>
+              <div class="hamburger-line"></div>
+              <div class="hamburger-line"></div>
+            </label>
+          </div>
+      </div>
+      <!-- Header tablet: logo in the middle, navbar on the sides-->
+      <div class="header-tablet">
+        <nav class="nav-tablet">
+          <a href="" class="nav-link">Home</a>
+          <a href="" class="nav-link">Highlights</a>
+          <img src="./Arctic-Logo.png" class="logo" alt="Logo">
+          <a href="" class="nav-link">News</a>
+          <a href="" class="nav-link">More</a> 
+        </nav>
+      </div>
+      <!-- Header desktop: logo on the left, navbar on the right-->
+      <div class= "header-desktop">
+          <img src="./Arctic-Logo.png" class="logo" alt="Logo">
+          <nav class="nav-desktop">
+            <a href="" class="nav-link">Home</a> |
+            <a href="" class="nav-link">Highlights</a> |
+            <a href="" class="nav-link">News</a> |
+            <a href="" class="nav-link">More</a> |
+          </nav>
```

Comment:
Instead of having three different `nav`s, you can have only one and then style it differently with media queries. Because the content stays the same, it's just the styling that changes

Reply:
Done :) Seemed like 3 navs was a better solution at the time :) I have a question about my dropdown menu (hamburger). Could it be repeating the text of the navs or should I somehow restyle my nav to be a part of drop down menu?

Reply:
If it's the same content, we should try to just style it differently with CSS instead of duplicating the HTML.